lib/phoenix_live_view/html_tokenizer.ex

```diff
@@ -59,124 +59,180 @@ defmodule Phoenix.LiveView.HTMLTokenizer do
     state = %{file: file, column_offset: indentation + 1, braces: [], context: []}
 
     case cont do
-      :text -> handle_text(text, line, column, [], tokens, state)
-      :style -> handle_style(text, line, column, [], tokens, state)
-      :script -> handle_script(text, line, column, [], tokens, state)
+      :text -> handle_text(text, {line, -column}, column, [], tokens, state)
+      :style -> handle_style(text, {line, -column}, column, [], tokens, state)
+      :script -> handle_script(text, {line, -column}, column, [], tokens, state)
       {:comment, _, _} -> handle_comment(text, line, column, [], tokens, state)
```

Comment:
I think we need to handle it here too otherwise the offset is off when we leave a comment.